translations/base-es.yaml

```diff
@@ -527,7 +527,7 @@ storyRewards:
 
     reward_painter:
         title: Pintor
-        desc: El <strong>pintor</strong> ha sido desbloqueado - ¡Extrae vetas de color (igual que lo haces con las figuras) y combínalas con una figura en el pintor para colorearlas! <br><br>PD: Si tienes alguna forma de daltonismo, ¡hay un <strong>modo para daltonicos</strong> en las configuraciones!
+        desc: El <strong>pintor</strong> ha sido desbloqueado - ¡Extrae vetas de color (igual que lo haces con las figuras) y combínalas con una figura en el pintor para colorearlas! <br><br>PD Si tienes alguna forma de daltonismo, ¡hay un <strong>modo para daltonicos</strong> en las configuraciones!
```

Comment:
Pretty sure that colon is supposed to be there, the string just needs to be escaped properly.

Reply:
Correct. Adjusted with proper escaped method.